### PR TITLE
Update docs with analytics info

### PR DESCRIPTION
**Pull request by ekinanp**
Signed-off-by: Enis Inan <enis.inan@puppet.com>

### CODE REVIEW

File: website/content/_index.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Wash is distributed as a single binary, and the only prerequisite is [`libfuse`]
 
 At this point, if you haven't already, you should start some resources that Wash can actually introspect. Otherwise, as Han Solo would say, "this is going to be a real short trip". So fire up some Docker containers, create some EC2 instances, toss some files into S3, launch a Kubernetes pod, etc. We've also provided a [guided tour](#wash-by-example) that includes a Docker application.
 
+**NOTE:** Wash collects data about how you use it. See the [analytics docs](/wash/docs#analytics) for more details.
```
Comment (MikaelSmith) [inline website/content/_index.md]: I'd say `collects anonymous data`.

File: website/content/docs/_index.md [hunk 2]
```diff
@@ -293,4 +297,28 @@ docker
...
+  * This includes any invocation of the `List`, `Exec`, `Read`, and `Stream` primitives
+  * Also includes the entry's plugin
+
+This data is associated with Bolt analytics' UUID (if available); otherwise, the data is associated with a random, non-identifiable user UUID.
+
+### Why does Wash collect data?
+Wash collects data to help us understand how it's being used and make decisions about how to improve it.
+
+### How can I opt out of Wash data collection?
+To disable the collection of analytics data add the following line to `~/.puppetlabs/wash/analytics.yaml`:
```
Comment (MikaelSmith) [inline website/content/docs/_index.md]: Should also reference the environment variable method.

**Comment by MikaelSmith** [comment]
I think you'll need to update your version of Hugo.

**Comment by ekinanp** [comment]
Updated.

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**